src/main/java/io/codelex/oop/persons/Customer.java

```diff
@@ -0,0 +1,25 @@
+package io.codelex.oop.persons;
+
+public class Customer extends Person {
+    private final String customerId;
+    private int purchaseCount = 0;
```

Comment:
primitīvajiem ir default value, dont need to assign in class scope

Reply:
Paldies! Izlaboju.